### DISCUSSION TITLE
Instalar version 0.7.0

### DISCUSSION THREAD

**Discussion by Saladito22** [kubilus1/autoortho#430] [General]
Estoy intentando instalar la version 7.0 lleva ya todo el dia y no se si se esta instalando o no.

**Comment by Jeff2122** [comment]
He escuchado esto antes: ¿puedes intentar usar v0.6.0 o v0.6.1 para la instalación y ver si es más rápido? Luego puedes cambiar a v0.7.0 si lo deseas.

**Reply by Saladito22** [reply]
Muchas Gracia s he intentado como me dices con la 6.0.1 y me pasa lo mismo los 15 zips los descarga mas o menos rapido pero luego en la descomprension es imposible Nose si es que
pongo alguna ruta mal, la primera casilla va a Custon Scenery la segunda a la raiz de x-plane12 y las otras dos donde vienen ac:

**Reply by Jeff2122** [reply]
Unpacking the zips takes a long time. You must be patient. If it hangs up during the install, it must be Windows security interfering, or something else. Always run the AO installer (if you use that method) and the exec as "Administrator" and make sure your target paths for the data and the downloads are present and writable for any user.
It might help to give the folders permissions for "Everyone". Easy to do - right click and in security tab, add "Everyone" to authorized users and give them read/write permissions. I have done this from the beginning of my time with AO, and have never had any of the issues people are reporting. I am not saying this is a fix, I am saying some people's Windows environments are more lenient than others when it comes to security. I suspect this, but I don't have proof.
I point my zip downloads to a junk folder on any drive, and point the data to a large SSD, and my XP12 folder is on a separate drive. I do not store anything or use AO from my system C:\ drive at all, except where AO keeps it's config data in the c:\Users folder, which is fine.

**Reply by Saladito22** [reply]
Vale voy a hacerlo de esa forma otorgando permisos a las carpetas y ya que el X-plane lo tengo instalado en un SSD externo instalo todo en el mismo sin tocar del la unidad C/: del sistema
Lo realizo desde el instalador de AO, no entiendo lo del ejecutivo como Administrador

**Reply by Jeff2122** [reply]
Run any AO executable as administrator. When you install it, and when you run the installed executable.

**Reply by Saladito22** [reply]
Buenos dias:
El sabado he empezado la instalación a las 10 de la mañana y los 15 zips se descargaron de una forma rapida, luego empezo lo que se supone que es la descomprension de los zips y la instalacion hay Domingo a  esta hora las 11 de la mañana esta en la linea
INFO: downloader:z_eur_09 is good. lo dejare hasta que supuestamente termine pero no me da muy buena impresion
Gracias

**Reply by Jeff2122** [reply]
I have no more ideas at this time other than this from my notes, some steps you have done already:
Check - v0.7.0 has very slow download speeds so some people are using v0.6.0 to do the download and install, then switching over to v0.7.0 to fly in the sim. 
Check - Always run as administrator.
Check - Make sure folder permissions are set for "Everyone" to read/write.
- Turn off Antivirus during install (use at your own risk)
- Turn off real-time scanning in Defender during install (use at your own risk)
- Internet connection and download speed doesn't have anything to do with the unpacking process. 
- Need more input on this problem

That is all I have so far. This issue crops up from time to time but I guess it gets solved and I miss the solution for my notes. Hopefully someone who solved the problem for themselves will see this and share the solution.
